routes/routes.celebrities.js

```diff
@@ -1,5 +1,35 @@
+const router = require("express").Router();
+const Celebrity = require('../models/Celebrity.model');
+
+
 // all your routes here
 
+router.get("/celebrities/create", (req,res,next) => {
+     Celebrity.create()
+    .then((celebArr) => {
+        res.render("celebrities/new-celebrity", celebArr)
+    })
+    .catch( err => console.log('there was an error ',err))
+})
```

Comment:
In this route we don't have any information yet, so all we want to do is render the form to create new celeb

```
router.get("/celebrities/create", (req,res,next) => {
    res.render("celebrities/new-celebrity");
})
```